all-actors/pom.xml

```diff
@@ -25,5 +25,10 @@
             <artifactId>certProcessor</artifactId>
             <version>1.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
```

Comment:
Log4j logger can be used instead of self4j

Reply:
done